setup.py

```diff
@@ -156,10 +157,21 @@ def prerelease_local_scheme(version):
 with open('README.md') as f:
     readme = f.read()
 
+
+def get_version(package):
+    """
+    Return package version as listed in `__version__` in `init.py`.
+    """
+    init_py = open(os.path.join(package, '__init__.py')).read()
+    return re.search("__version__ = ['\"]([^'\"]+)['\"]", init_py).group(1)
+
+version = get_version('s3_file_field')
```

Comment:
This entire code block will be non-functional. The `version` being passed here is ignored. The option `use_scm_version` overwrites the version dynamically.